### ISSUE TITLE
Fix emissions module xmls

### ISSUE THREAD

**Issue by kdorheim** [JGCRI/gcamdata#883]
As part of #815 we came across a few problems with emissions module xml files.

The new data system is missing 4 xml files. 
There are discrepancies in the `all_fgas_emissions.xml`

Please see the attached zip file for the following,

* a html summarizing the emissions-xml problems and relevant data system chunk information
the old and new xml files that fail the xml test
* a txt file of the complete xml test output
* the old and new `all_fgas_emissions.xml` files 
* processing code from the old data system used to create the `all_fgas_emissions.xml`

[fix_emissions_xmls_materials.zip](https://github.com/JGCRI/gcamdata/files/1894842/fix_emissions_xmls_materials.zip) | old

**Comment by kdorheim** [comment]
@cahartin with #934 3/4 missing xmls are created without discrepancies! 

The emissions module is still missing 1 xml and has a discrepancy in one file, these problems were expected.  I have attached the updated fix emissions xml materials. 

```
21 files tested.
1 missing output file.
1 file discrepancy.
```

**Comment by cahartin** [comment]
wohoo! i'm hoping to tackle the last missing xml.

**Comment by kdorheim** [comment]
This issue is closed by #946  
```
21 files tested.
0 missing output files.
0 file discrepancies.
```
Thanks @cahartin and @russellhz  for helping out!!
